manual/guides/uml.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: manual
+title: Generating UML Diagrams
+subtitle: "Generating UML Diagrams"
```

Comment:
This is a little redundant.
![image](https://github.com/user-attachments/assets/549cbf21-6ee7-4285-a737-03821fd2ad1f)
```suggestion
title: Inheritance Diagrams
subtitle: "Generating Inheritance Diagrams with UML"
```